src/lib/Containers/__tests__/Inbox-tests.tsx

```diff
@@ -1,154 +1,172 @@
 import * as React from "react"
+import * as TestUtils from "react-dom/test-utils"
 import "react-native"
 import * as renderer from "react-test-renderer"
 
+import ZeroStateInbox from "../../Components/Inbox/Conversations/ZerostateInbox"
 import Inbox from "../Inbox"
 
 it("renders correctly", () => {
-  const tree = renderer.create(<Inbox me={meProps} />).toJSON()
+  const tree = renderer.create(<Inbox me={meProps()} />).toJSON()
   expect(tree).toMatchSnapshot()
 })
 
-const meProps = {
-  conversations: {
-    edges: [
-      {
-        node: {
-          id: "582",
-          inquiry_id: "59302144275b244a81d0f9c6",
-          from: { name: "Jean-Luc Collecteur", email: "luc+messaging@artsymail.com" },
-          to: { name: "ACA Galleries" },
-          last_message: "Karl and Anna... Fab!",
-          created_at: "2017-06-01T14:14:35.538Z",
-          artworks: [
-            {
-              id: "bradley-theodore-karl-and-anna-face-off-diptych",
-              href: "/artwork/bradley-theodore-karl-and-anna-face-off-diptych",
-              title: "Karl and Anna Face Off (Diptych)",
-              date: "2016",
-              artist_names: "Bradley Theodore",
-              image: {
-                url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/normalized.jpg",
-                image_url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/:version.jpg",
-              },
+it("shows empty state if there's no data", () => {
+  const tree = renderer.create(<Inbox me={meProps(false, false)} />).toJSON()
+  // const emptyStateView = TestUtils.scryRenderedComponentsWithType(tree, ZeroStateInbox)
```

Comment:
After trying several calls, I can't get `TestUtils` to return a particular child component. And unfortunately we can't use `enzyme` with our current RN version so I'll leave it commented for now.